testfiles/CMakeLists.txt

```diff
@@ -32,6 +32,15 @@ ADD_SIMULATION_TEST(IDF_FILE 5ZoneAirCooled_UniformLoading.idf EPW_FILE USA_IL_C
 ADD_SIMULATION_TEST(IDF_FILE 5ZoneAirCooled_VRPSizing.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE 5ZoneAirCooled_VRPSizing_MaxZd.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
 ADD_SIMULATION_TEST(IDF_FILE 5ZoneAirCooled_ZoneAirMassFlowBalance.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboard.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardConvLoad.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardMRT.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardODB.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardOpTemp.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardOWB.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardSimpleOff.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardTotalLoad.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
+ADD_SIMULATION_TEST(IDF_FILE 5ZoneCoolingPanelBaseboardVarOff.idf EPW_FILE USA_IL_Chicago-OHare.Intl.AP.725300_TMY3.epw)
```

Comment:
@mjwitte Well, it wouldn't be "easy" because each file has a cooling panel in only one of the systems.  I did that so that I could run each of the nine files and compare the results to each other to see what the cooling panel was doing in each case knowing that I was comparing the same zone across files.  Changing this would require me to go in and modify at least two more HVAC systems in the input file and I have to admit that could get a bit messy.  Can we say that three is a big improvement over nine and that the addition of a unit test was a nice side benefit? :)